### ISSUE TITLE
Random Nitpicks

### ISSUE THREAD

**Issue by TehNut** [disconsented/ANSSRPG#26]
- [x] Upgradle. pls.

- [x] Get everything to a standardized formatting. There are many instances of the following all over the place:

```
    public void someMethod() {
        // Thing
    }
```

```
    public void someMethod(){
        // Thing
    }
```

```
    public void someMethod() 
    {
        // Thing
    }
```

- [x] Fix the `/help` command. Very easy to do. Just change `getCommandUsage` in `Perks.java` and `Skills.java` command classes from returning `null` to `""`.

- [x] Move away from `System.out.println()`. It gives the user absolutely no information about where the console message came from. You can easily make a logger with `public static Logger logger = LogManager.getLogger(LOGGERNAME);`. May I suggest just using [this](http://puu.sh/fslnX/632d936bd9.txt)?

- [x] Move basic mod information from `build.gradle` to a `gradle.properties` file.

- [x] Move basic mod information in `Main.java` to variables either in `Main.java` or in another class. IE: `public static final String NAME = "A Not So Simple RPG";` You can find an example of that in my project [here](https://github.com/TehNut/ResourcefulCrops/tree/master/src/main/java/tehnut/resourcecrops). Check both `ResourcefulCrops.java` and `ModInformation.java`. 

I probably have more, but it's 2am and I can't think clearly. If you would like help with / would prefer somebody else do any of these, I can PR them in.

**Comment by TehNut** [comment]
Checkmarks 1 and 5 would be done by merging #27.

**Comment by TehNut** [comment]
Closed by 0ac47168483ba2f71d7f9a83d4368c54ea0a14f3
